blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -3,19 +3,23 @@ import Immutable from 'immutable';
 since we would return a function, */
```

Comment:
There was a note here made about not being able to extend Record - I'm curious if anyone knows what the problem was?